internal/policy/verify.go

```diff
@@ -410,6 +410,23 @@ func VerifyTag(ctx context.Context, repo *git.Repository, ids []string) map[stri
 			continue
 		}
 
+		tagRef, err := repo.Tag(strings.Trim(absPath, "refs/tags"))
+		if err != nil {
+			status[id] = err.Error()
+			continue
+		}
+
+		tagObj, err := gitinterface.GetTag(repo, tagRef.Hash())
+
+		if err != nil {
+			status[id] = err.Error()
+			continue
+		}
+
+		if entry.TargetID != tagObj.Target {
```

Comment:
is this right? the entry may point to the tag obj, not the target of the tag obj

Reply:
I have changed this to check if it is either pointing to the entry.TargetID, or the tagObj itself